doc/neps/roadmap.rst

```diff
@@ -51,6 +49,8 @@ Other performance improvement ideas include:
 
 - A better story around parallel execution (related is support for free-threaded
   CPython, see further down).
+- Using recent improvements in our ufunc context to allow users for example to
```

Comment:
Can you rephrase this slightly, removing "our" and perhaps pointing the API that is meant here?

Reply:
I added ``errstate`` in brackets, which is what I mean (mostly).  Since I think that is a path towards switching ufunc loops, but of course I also mean the new C API which allows us to actually propagate that information.